js/viz/gauges/utils/resolve_label_overlapping.js

```diff
@@ -0,0 +1,104 @@
+const ARC_COORD_PREC = 5;
+
+import { normalizeAngle, normalizeArcParams } from '../../core/utils';
+
+export function dividePoints(bars) {
+    return bars.reduce(function(r, bar) {
+        const angle = normalizeAngle(bar._angle);
+        bar._text._lastCoords = { x: 0, y: 0 };
+        (angle <= 90 || angle >= 270 ? r.right : r.left)
```

Comment:
Let's save this value to a separate variable to make code clearer

Reply:
I put the value in a variable